ASI.Basecode.Services/Services/UserService.cs

```diff
@@ -44,6 +44,11 @@ public void AddUser(UserViewModel model)
                 user.CreatedBy = System.Environment.UserName;
                 user.UpdatedBy = System.Environment.UserName;
 
+                Guid userId = Guid.NewGuid();
+                user.UserId = userId.ToString();
+                user.RoleId = 1;
+                user.Name = model.Name;
+                user.Email = "Jermain@buang.com";
```

Comment:
Please do change when the user registration is ready. The data must be comming from the model.